src/utils.js

```diff
@@ -185,11 +184,9 @@ async function loadConfig(loaderContext, config, postcssOptions) {
       options: postcssOptions || {},
     };
 
-    result.config = result.config(api);
+    return { ...result, config: result.config(api) };
```

Comment:
Very uninformed questions, but this is not the same as a deep clone :
- Will it not create issue ? 
- Was a deep clone here an overkill in the first place ?

Thank you for your time ! You are a pillar of the Webpack ecosystem !

Reply:
Ok thank you so mush for the details and your time. Have a nice day !